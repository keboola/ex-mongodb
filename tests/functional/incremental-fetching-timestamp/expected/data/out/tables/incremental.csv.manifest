{"primary_key":null,"incremental":true}