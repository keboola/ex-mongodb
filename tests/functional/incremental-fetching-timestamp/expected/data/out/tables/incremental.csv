"4","283.473","2020-04-18T16:00:00Z","1587146020"
"3","783.028","2020-05-18T11:00:00Z","1587606020"
"2","133.444","2020-02-15T13:00:00Z","1587626020"
"1","123.344","2020-05-18T16:00:00Z","1587646020"
